repository/src/main/java/org/alfresco/repo/content/transform/TransformerDebug.java

```diff
@@ -937,20 +863,45 @@ private String getRenditionName(String renditionName)
     /**
      * Debugs a response to the Transform Service
      */

```

Comment:
Needed to split this method into two parts, so that the enterprise-repo can insert debug from the t-router between the two. That way we see the transforms that take place in the t-router, metadata extractions and then the "Finished in xx ms" message.